packages/android_intent_plus/android/src/main/java/dev/fluttercommunity/plus/androidintent/MethodCallHandlerImpl.java

```diff
@@ -92,11 +92,17 @@ public void onMethodCall(@NonNull MethodCall call, @NonNull Result result) {
             action, flags, category, data, arguments, packageName, componentName, type);
 
     if ("launch".equalsIgnoreCase(call.method)) {
-      sender.send(intent);
 
+      if (intent != null && !sender.canResolveActivity(intent)) {
```

Comment:
I have just moved this check from `buildIntent()` to be able to set package as `null`, so the system just opens a chooser.